test-case-reporting/src/test_result_record.ts

```diff
@@ -273,4 +274,22 @@ export class TestResultRecord {
 
     return this.bigJoinQuery(queryFunction, pageInfo);
   }
+
+  /**
+   * Gets a list of the most recently uploaded builds
+   * @param pageInfo object with the limit and the offset of the query
+   */
+  async getRecentBuilds({limit, offset}: PageInfo): Promise<Array<Build>> {
+    const dbBuilds = await this.db<DB.Build>('builds')
+      .orderBy('started_at', 'DESC')
+      .limit(limit)
+      .offset(offset);
+
+    /* eslint-disable @typescript-eslint/camelcase */
+    return dbBuilds.map(({commit_sha, build_number}) => ({
+      commitSha: commit_sha,
+      buildNumber: build_number,
```

Comment:
Is the pass/fail status of a build available?

Reply:
Not at the moment. To know the actual pass/fail/err status of the build, we would need to call the Travis API.

We could estimate the status by checking whether a test failure was reported for the build, which I think would be more useful in our application. What do you think?

Reply:
I agree that would be useful, albeit tricky to query efficiently. I'd say let's punt on build status for now to avoid that complexity, and it can be looked at later if needed.

Reply:
I'll note that at the time a Job is being inserted, we probably have the info necessary to report passed/failed. A job list may end up being more useful than a build list, but again for now I think punting on this is fine.